app/models/manageiq/providers/google/cloud_manager/provision_workflow.rb

```diff
@@ -0,0 +1,23 @@
+class ManageIQ::Providers::Google::CloudManager::ProvisionWorkflow < ::MiqProvisionCloudWorkflow
+  def allowed_instance_types(_options = {})
+    source = load_ar_obj(get_source_vm)
+    ems = get_targets_for_ems(source, :cloud_filter, Flavor, 'flavors')
+    ems.each_with_object({}) { |f, h| h[f.id] = display_name_for_name_description(f) }
+  end
+
+  def allowed_disk_sizes(_options = {})
+    min = 10
+    max = 10 * 1024 # 10 TB
+    (min..max).each_with_object({}) { |i, h| h[i] = i.to_s }
```

Comment:
Yes, I meant to ask, this seemed excessive but I couldn't find a good alternative in the existing code. By adjust the values, do mean the full range of memory sizes is available to users, or just prescribed ones in the dialog?